deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -4,6 +4,11 @@ metadata:
   annotations:
     alm-examples: '[]'
     capabilities: Basic Install
+    categories: OpenShift Optional
+    certified: "false"
+    containerImage: REPLACE_IMAGE
```

Comment:
Should a placeholder be added for the version as well?
`  name: windows-machine-config-operator.v0.0.0`

Reply:
I thought we are doing this and it is being replaced in `render_templates`. Or are you asking about adding `name` to `annotations`.

Reply:
@mansikulkarni96 please confirm the `REPLACE_IMAGE` works with `render_templates` before we merge.

Reply:
yes it works and we do have the name placeholder just below the annoations.